resources/js/components/User/mixins/InteractWithUser.js

```diff
@@ -111,7 +111,7 @@ export default {
     },
 
     created() {
-        onOnce('logout', this.onLogout)
+        onOnce(this, 'logout', this.onLogout)
```

Comment:
Maybe pass `this.$root` from here so we don't have to append `.$root` in the onOnce function.
This will allow the function to be used on components as well instead of just the root.

Still odd that it would be necessary though considering vues composition api and vueuse can interact with the vue container without issue

Reply:
> Still odd that it would be necessary though considering vues composition api and vueuse can interact with the vue container without issue

Yeah the order of operations here is weird. Something about how mixins are fully created and mounted before the Vue object is created. I debugged the actual vue code for a while and there really isn't a clean way around it without writing these kinds of mixins differently, which is a much bigger and possibly breaking update.